pingu/corpus/assets/label.py

```diff
@@ -13,13 +13,15 @@ class Label(object):
         start (float): Start of the label within the utterance in seconds. (default: 0)
         end (float): End of the label within the utterance in seconds. (default: -1) (-1 defines
                      the end of the utterance)
+        label_list (LabelList): The label-list this label is belonging to.
     """
-    __slots__ = ['value', 'start', 'end']
+    __slots__ = ['value', 'start', 'end', 'label_list']
 
     def __init__(self, value, start=0, end=-1):
         self.value = value
         self.start = start
         self.end = end
+        self.label_list = None
 
     def __eq__(self, other):
         return (self.start, self.end, self.value.lower()) == (other.start, other.end, other.value.lower())
```

Comment:
The test should make clear that we're ignoring the referenced label list on purpose and that two labels are considered the same even the label lists are different. Same applies to `__lt__`.